warehouse/oidc/models/github.py

```diff
@@ -269,7 +269,6 @@ def reify(self, session):
         Returns a `GitHubPublisher` for this `PendingGitHubPublisher`,
         deleting the `PendingGitHubPublisher` in the process.
         """
-
```

Comment:
Stray whitespace removal here!

Reply:
Doh!  I thought I got this already.  Fixing.